### ISSUE TITLE
latexmk: bad command line argument

### ISSUE THREAD

**Issue by frankylwf** [thomasjo/atom-latex#366]
error message: 
latexmk: bad command line argument
Parsing of log files failed

using texlive 2016 in win10 x64.
my Tex path is C:\texlive\2016\bin\win32
my output path is C:\Latex

Latexmk version:
 Latexmk, John Collins, 19 Jan. 2017. Version 4.52c

**Comment by yitzchak** [comment]
Where is your source file located?

**Comment by frankylwf** [comment]
texlive: C:\texlive\2016\bin\win32

**Comment by yitzchak** [comment]
Sorry, I meant, where is you LaTeX document located?

**Comment by frankylwf** [comment]
My document first.tex is located here:
C:\Users\frank\OneDrive\Document\LaTeX

And i have tried "latexmk -pdf fisrt.tex" with no error and the log file has no warning.

**Comment by yitzchak** [comment]
Can you try changing your output directory a relative directory like `build`?

**Comment by frankylwf** [comment]
I tried with "latex-output", but the same error

**Comment by yitzchak** [comment]
Can you give me a rundown of the settings from the `latex` package please.

**Comment by frankylwf** [comment]
![1](https://cloud.githubusercontent.com/assets/28207430/25567187/a50a4458-2de8-11e7-8174-0a83687a50ba.PNG)
![2](https://cloud.githubusercontent.com/assets/28207430/25567188/a50bd7aa-2de8-11e7-8c9c-f30f69b478ca.PNG)
![3](https://cloud.githubusercontent.com/assets/28207430/25567189/a50c4686-2de8-11e7-8ead-c649ef0ef07c.PNG)

**Comment by yitzchak** [comment]
I'll see if I can reproduce using your settings. Until then you might try again with the "Extended Build Mode" off.

**Comment by frankylwf** [comment]
I have just re-installed win10. Can you tell me that MiKTex and TexLive, which is better for atom? And should I install others software?

**Comment by yitzchak** [comment]
TL;DR Both should work with Atom, but I prefer TeX Live.

In my opinion, it includes more supporting software and more up to date. Please note that TeX Live does not install packages on demand like MiKTeX does, but installing packages with package manager is not difficult. There is a package in TeX Live to do this, but given the size of hard drives I just do a full install.

**Comment by frankylwf** [comment]
Thank you, and is that mean I can install TexLive 2016 by default?

**Comment by yitzchak** [comment]
I use TeX Live 2016 by default with no current issues.

**Comment by frankylwf** [comment]
When I intall Tex Live 2016, should I install Texworks by the guide of installation?

**Comment by yitzchak** [comment]
It shouldn't matter. TeX works is an alternate editor. If you are going to use Atom you don't need it.

**Comment by frankylwf** [comment]
After re-installed, there is another error: texification failed with status code 3221225504

**Comment by frankylwf** [comment]
I uninstall the Tex Live and re-install  again, it works, but I don't know why.

**Comment by yitzchak** [comment]
Glad you got it working!

I believe that status code is STATUS_INVALID_FILE_FOR_SECTION (0xC0000020) which is thrown by a Windows device driver. You might want to run a disk scan/check and do an antivirus check.

Closing, let us know if you have further problems.
